lib/model.dart

```diff
@@ -10,19 +10,19 @@ class Person {
   num age;
   String age_category;
   String gender;
-  String IDPStatus;
+  String idp_status;
```

Comment:
idpStatus would be conventional in Dart?

Reply:
Fixed